haplotypecaller-gvcf-gatk4.wdl

```diff
@@ -163,11 +166,13 @@ task HaplotypeCaller {
     File input_bam_index
     File interval_list
     String output_filename
+    String vcf_basename
```

Comment:
Why don't we derive this from output_file name?

Reply:
vcf_basename was already created within the [workflow block](https://github.com/gatk-workflows/gatk4-germline-snps-indels/blob/c07d40a1ef5f5116a9ee6734cf3e7a5e1d17bb9c/haplotypecaller-gvcf-gatk4.wdl#L55) so I thought it would be simpler to use the same variable. But i'll edit this to move the vcf_basename to outside the task input block and set it with  `basename(output_filename)` since output_file name has the vcf/gvcf extension